FILE packages/desktop/src-tauri/Info.plist
RANGE {"path":"packages/desktop/src-tauri/Info.plist","old_lines":[3,6],"new_lines":[3,8],"diff_block_lines":[5,11],"hunk_header":"@@ -3,4 +3,6 @@","substantive_changed_lines":2}
@@ -3,4 +3,6 @@
 <plist version="1.0">
   <dict>
+    <key>ApplePressAndHoldEnabled</key>
+    <false/>
     <key>CFBundleIdentifier</key>
     <string>dev.paseo.desktop</string>
